### PR TITLE
[docs] Show how to use `sui client faucet` for getting tokens from faucet

### PR DESCRIPTION
**Pull request by stefan-mysten**
## Description 

This PR updates the docs to show how to to use `sui client faucet` for getting tokens from faucet.

## Test Plan 

Docs, no tests needed.

---
If your changes are not user-facing and do not break anything, you can skip the following section. Otherwise, please briefly describe what has changed under the Release Notes section.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### Release notes
This PR updates the docs to show how to to use `sui client faucet` for getting tokens from faucet.

### CODE REVIEW

File: docs/content/guides/developer/getting-started/local-network.mdx [hunk 1]
```diff
@@ -108,52 +108,9 @@ Use the active address to get test SUI to use on your local network. Use the `su
...
-        {
-            "amount": 200000000,
-            "id": "0xd9f0b521443d66227eddc2aac2e16f667ca9caeef9f1b7afb4a6c2fc7dcb58d8",
-            "transferTxDigest": "7sp4fFPH2WaUgvN43kjDzCpEhKfifqjx5RTki74y8T3E"
-        }
-    ],
-    "error": null
-}
-```
+In the most simple case, run `sui client faucet` and wait up to 60s for your tokens to reach your wallet. Use `sui client gas` to check for the new tokens.
```
Comment (ronny-mysten) [inline docs/content/guides/developer/getting-started/local-network.mdx:113]:
```suggestion
In the most basic case, run `sui client faucet` and wait up to 60 seconds for the coins to reach your wallet. Use `sui client gas` to check for the new coins.
```
The coin/token wording has changed with the intro of Closed Loop Token standard. I'll make it consistent across the site in separate PR.

File: docs/content/guides/developer/getting-started/local-network.mdx [hunk 2]
```diff
@@ -108,52 +108,9 @@ Use the active address to get test SUI to use on your local network. Use the `su
 
 Transactions on your local network require SUI coins to pay for gas fees just like other networks. To send coins to a Sui Wallet connected to your local network, see [Set up a local Sui Wallet](#set-up-a-local-sui-wallet). You can use the address for the local Sui Wallet with the faucet.
 
-Use the following cURL command to get test coins from the local faucet.
+Sui CLI provides the `sui client faucet` command to get tokens from the faucet. If a different network than the usual ones (fullnode.network.sui.io) is used, it will complain and ask to pass the url to the faucet server. By default, it will use the active address and the active network environment. If you need to pass in a different address or faucet server url, check the help menu.
```
Comment (ronny-mysten) [inline docs/content/guides/developer/getting-started/local-network.mdx:111]:
```suggestion
Sui CLI provides the `sui client faucet` command to get coins from the faucet. If you're using a different network than the public ones (fullnode.network.sui.io), you receive a prompt to pass the URL to the faucet server. The `faucet` command uses the active address and the active network environment by default. If you need to pass in a different address or faucet server URL, check the `help` menu.
```

**Review by ronny-mysten — APPROVED** [review]
Approving to carry forward. Thanks!

**Comment by stefan-mysten** [comment]
@ronny-mysten thanks, fixed as per your suggestions.

**Pull request merged**